server/src/main/java/org/opensearch/extensions/rest/RestSendToExtensionAction.java

```diff
@@ -170,7 +185,7 @@ public String executor() {
                 ExtensionsManager.REQUEST_REST_EXECUTE_ON_EXTENSION_ACTION,
                 // HERE BE DRAGONS - DO NOT INCLUDE HEADERS
```

Comment:
Let's update this comment to "DO NOT INCLUDE HEADERS WITH SECURITY OR PRIVACY INFORMATION".

And it might be good to move the allowlist/denylist code above down here. I know it doesn't need to be in a catch block, but I think it will be more readable right next to this big warning.

Reply:
I should put allowlist/denylist code above transportService.sendRequest call?